app/code/community/Ebizmarts/MailChimp/Model/Api/Carts.php

```diff
@@ -402,100 +406,105 @@ protected function getBatchLimitFromConfig()
     protected function _getCustomer($cart, $mailchimpStoreId, $magentoStoreId)
     {
         $helper = $this->getHelper();
-        $api = $helper->getApi($magentoStoreId);
-        if ($cart->getCustomerId()) {
-            try {
-                $customer = $api->ecommerce->customers->get($mailchimpStoreId, $cart->getCustomerId(), 'email_address');
-            } catch (MailChimp_Error $e) {
-                $err = $e->getMailchimpTitle();
-                if (!preg_match('/Resource Not Found for Api Call/', $err)) {
-                    $msg = "Failed to lookup e-commerce customer via ID " . $cart->getCustomerId();
-                    $helper->logError($msg . ': ' . $e->getFriendlyMessage(), $magentoStoreId);
+        $customer = array();
+        try {
```

Comment:
Move the `try/catch` for `getApi` to a separate block (like in app/code/community/Ebizmarts/MailChimp/Model/Api/Orders.php line 249), to avoid nested `try/catch` blocks.